### DISCUSSION TITLE
How to change title color when tab is unfocused

### DISCUSSION THREAD

**Discussion by OlaoluwaM** [Eugeny/tabby#8420] [Q&A] [answered]
I'm looking to modify the color of a tab's title when it's unfocused. Any ideas? Also, since this is an electron app, is it possible to open up the developer console?

**Comment by Eugeny** [comment, marked as answer]
Yes, you can modify it through custom CSS (in Settings). The developer console is available through Settings -> Application -> Debugging

**Comment by lotep** [comment]
I think the ACTIVE connection shoud have a bright color and all other tabs should be gray (or something like that!).
This way woudl be easier to see whats the active tab.
